tests/cypress/integration/actions_users/registration_involved/case_28_review_pipeline_feature.js

```diff
@@ -365,10 +365,10 @@ context('Review pipeline feature', () => {
 
         it('Issue navigation. Navigation works and go only to frames with issues.', () => {
             cy.get('.cvat-issues-sidebar-previous-frame').should('have.attr', 'style').and('contain', 'opacity: 0.5;'); // The element is not active
-            cy.get('.cvat-issues-sidebar-next-frame').should('not.have.css', 'pointer-events', 'none').click();
+            cy.get('.cvat-issues-sidebar-next-frame').trigger('mouseout').click();
```

Comment:
Could you please explain?

Reply:
As an additional action before the click. This could potentially help for interacting with the button.

Reply:
I hardly think so, but ok, let's look